src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.tsx

```diff
@@ -20,13 +20,21 @@ import type {
 } from 'signals/incident/components/form/MapSelectors/types'
 import { selectionIsObject } from 'signals/incident/components/form/MapSelectors/constants'
 
+const ICON_SIZE = 40
 const mapWidth = 640
 const mapHeight = 300
 const mapZoom = 12
 
-const Address = styled.address`
-  margin-bottom: ${themeSpacing(4)};
-  font-style: normal;
+const LocationDescription = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
No need to define `row`; this is the default value

Reply:
✅